### ISSUE TITLE
Build info unknown when using docker image

### ISSUE THREAD

**Issue by fmassot** [quickwit-oss/quickwit#2288]
When using a docker image, here is the json returned by the API `api/v1/version`

```json
{
  "commit_version_tag": "none",
  "cargo_pkg_version": "0.3.1",
  "cargo_build_target": "x86_64-unknown-linux-gnu",
  "commit_short_hash": "unknown",
  "commit_date": "unknown",
  "version": "0.3.1-nightly"
}
```

**Comment by fmassot** [comment]
We obviously show "unknow" when starting quickwit too.

**Comment by guilload** [comment]
Closed via #2299.
